docs/projects/ci-integration.md

```diff
@@ -81,3 +82,78 @@ run:
 ```
 
 See the full example in [**https://gitlab.com/yhaviv/test2**](https://gitlab.com/yhaviv/test2)
+
+## Using Jenkins Pipeline
+
+When using [Jenkins Pipeline](https://www.jenkins.io/doc/book/pipeline/) you need to set up the credentials/secrets in Jenkins and 
+and update the script `Jenkinsfile` in your codebase. You can trigger the Jenkins pipeline either through Jenkins triggers or through the GitHub webhooks. 
+
+Example `Jenkinesfile` that is invoked when you start a Jenkins pipeline (via triggers or GitHub webhooks):
+
+```Groovy
+pipeline {
+   agent any
+    environment {
+      RELEASE='1.0.0'
+      PROJECT_NAME='project-demo'
+    }
+   stages {
+      stage('Audit tools') {
+         steps{
+            auditTools()
+         }
+      }
+      stage('Build') {
+            environment {
+               MLRUN_DBPATH='https://mlrun-api.default-tenant.app.us-sales-341.iguazio-cd1.com'
+               V3IO_ACCESS_KEY=credentials('V3IO_ACCESS_KEY')
+               V3IO_USERNAME='xingsheng'
+               V3IO_API="https://default-tenant.app.us-sales-341.iguazio-cd1.com:8444"
```

Comment:
in new ver (>= 1.0) V3IO_API is determined from the `MLRUN_DBPATH` , can you try running it without setting the `V3IO_API`

Reply:
yes, it can run w/o setting the V3IO_API